src/calculateRentalCost.js

```diff
@@ -6,5 +1,22 @@
+const LONG_TERM = 3;
+const BASEPRISE = 40;
+const SELL = 20;
+
+const GOOD_DAYS = 7;
+const DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const fullCost = days * BASEPRISE;
```

Comment:
The calculation of `fullCost` is correct, but ensure that the variable `BASEPRISE` is correctly named as `BASEPRICE`.